src/main/cli/settings.c

```diff
@@ -973,6 +973,7 @@ const clivalue_t valueTable[] = {
     { "servo_lowpass_hz",           VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { 0, 400}, PG_SERVO_CONFIG, offsetof(servoConfig_t, servo_lowpass_freq) },
     { "tri_unarmed_servo",          VAR_INT8   | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_SERVO_CONFIG, offsetof(servoConfig_t, tri_unarmed_servo) },
     { "channel_forwarding_start",   VAR_UINT8  | MASTER_VALUE, .config.minmaxUnsigned = { AUX1, MAX_SUPPORTED_RC_CHANNEL_COUNT }, PG_SERVO_CONFIG, offsetof(servoConfig_t, channelForwardingStartChannel) },
+    { "servo_custom_rate",          VAR_INT8   | MASTER_VALUE, .config.minmax = { -SERVO_RATE_MAX, SERVO_RATE_MAX }, PG_SERVO_CONFIG, offsetof(servoConfig_t, servo_custom_rate) },
```

Comment:
custom rate is common for all servos (except direction). It will (probably) interfere with servo gimbal